pyproject.toml

```diff
@@ -2,5 +2,35 @@
 requires = ["setuptools>=40.8.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
+
+[project]
+name = "armicontrib-openmc"
+version = "1.1.0"
+description = "ARMI plugin for neutronics analysis with OpenMC."
+license = {file = "LICENSE.md"}
+requires-python = ">3.6"
+readme = "README.rst"
```

Comment:
I think this should be: `README.md`

Reply:
Absolutely, good catch. Fix has been pushed